pkg/descheduler/policyconfig.go

```diff
@@ -62,21 +62,22 @@ func decode(policyConfigFile string, policy []byte, client clientset.Interface,
 	if err != nil {
 		return nil, err
 	}
-
-	setDefaults(*internalPolicy, registry, client)
-
-	return internalPolicy, nil
+	return setDefaults(*internalPolicy, registry, client)
 }
 
-func setDefaults(in api.DeschedulerPolicy, registry pluginregistry.Registry, client clientset.Interface) *api.DeschedulerPolicy {
```

Comment:
Worth changing `in api.DeschedulerPolicy` into `in *api.DeschedulerPolicy` to avoid reference -> dereference steps.